formidable/forms/validations/presets.py

```diff
@@ -179,6 +203,14 @@ class ConfirmationPresets(Presets):
     description = _("Ensure both fields are identical")
     default_message = _("{left} is not equal to {right}")
 
+    def __init__(self, arguments, fields=None, message=None):
+        super(ConfirmationPresets, self).__init__(
+            arguments, fields=fields, message=message)
+        right_arg = self.arguments['right']
+        if not right_arg.field_id and fields:
```

Comment:
do you believe `cast_value_with` should be used on every arguments (field+values) or only on values ?